### PR TITLE
ref(electron): Update Electron integration code

### PR DESCRIPTION
**Pull request by jan-auer**
Updated integration code example due to https://github.com/getsentry/sentry-electron/pull/53:

- [x] Update code example with new syntax
- [x] Use public DSN instead of private one
- [x] Do not require `electron-download`
- [x] Update link to docs (moved to `/clients/electron`)

screen shot 2018-04-09 at 09 17 22

### CODE REVIEW

**Comment by jan-auer** [comment]
**HOLD**, this needs more fixes...

File: lib/Steps/Integrations/Electron.ts [hunk 1]
```diff
@@ -4,17 +4,37 @@ import * as _ from 'lodash';
...
-// tslint:disable-next-line:radix
 const MIN_ELECTRON_VERSION = parseInt(
   MIN_ELECTRON_VERSION_STRING.replace(/\D+/g, ''),
+  10,
 );
+
+const CODE_EXAMPLE = `const Sentry = require('@sentry/electron');
+
+Sentry.init({
+  dsn: '___DSN___',
```
Comment (kamilogorek) [inline lib/Steps/Integrations/Electron.ts]: Should we want use `___PUBLIC_DSN___` here as well?

**Comment by kamilogorek** [comment]
Just this one comment. Other than this, its ready to go 👍

File: lib/Steps/Integrations/Electron.ts (hunk 1 above)
Comment (jan-auer) [reply lib/Steps/Integrations/Electron.ts]: It's not replaced by the same mechanism, and TIL `___DSN___` will be changed to emit the public DSN soon, so I'm gonna leave it for brevity.

**Pull request merged**